src/js/menu/menu-button.js

```diff
@@ -37,6 +37,16 @@ class MenuButton extends ClickableComponent {
   }
 
   /**
+   * Hide the menu if the number of items is less than or equal to this threshold. This defaults
+   * to 1 beacuse all menus have a title.
```

Comment:
Not all menus have a title - the Subtitles menu and the Descriptions menu have an "off" button, which should be hidden if it is the only item. <Captain Pedantic ;-) >